crane-cloud/frontend
# Feat View all your projects as a User

#### What does this PR do?

Vies your projects

#### Description of Task to be completed?
- Display a users specific projects

#### How should this be manually tested?

* Check it out here.
**NOTE that this PR makes use of functionality in the auth PR to pick user ID to add it to the backend route for the feature. So to perfectly run it, FOR NOW, you will have to manually put the user ID in the route to see the feature working perfectly**

#### Screenshots

![image](https://user-images.githubusercontent.com/32802973/76400444-6094e580-6391-11ea-9744-c295fffb9561.png)
